mukunzidd/project11
# Pacth API Homework

### What does this PR do?
This pull request set up the patch method for editing todos

### Description of Task to be completed?
The task done here are
 - function that update each property of the todo object.

### How should this be manually tested?
Merge the PR to your dev path and you can see the changes.
Send a patch request to
```http://127.0.0.1:3000/todos/id```
